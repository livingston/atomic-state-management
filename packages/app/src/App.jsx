function App() {
  return (
    <>
      <header>
        <nav className="bg-gray-800 shadow">
          <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
            <div className="flex items-center justify-between h-16">
              <div className="flex items-center">
                <div className="flex-shrink-0"></div>
              </div>
              <div className="hidden md:block">
                <div className="ml-10 flex items-baseline space-x-4"></div>
              </div>
            </div>
            <div className="flex items-center"></div>
          </div>
        </nav>
      </header>
      <main>
        <div className="max-w-7xl mx-auto px-4 py-6 sm:px-0">
          <div className="border-1 border-gray-400 rounded-lg px-10 py-10 bg-gray-100 shadow"></div>
        </div>
      </main>
    </>
  );
}

export default App;
