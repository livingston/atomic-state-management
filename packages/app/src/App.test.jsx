import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
  it('should render the component', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
