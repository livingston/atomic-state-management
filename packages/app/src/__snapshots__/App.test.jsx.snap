// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the component 1`] = `
<div>
  <header>
    <nav
      class="bg-gray-800 shadow"
    >
      <div
        class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8"
      >
        <div
          class="flex items-center justify-between h-16"
        >
          <div
            class="flex items-center"
          >
            <div
              class="flex-shrink-0"
            />
          </div>
          <div
            class="hidden md:block"
          >
            <div
              class="ml-10 flex items-baseline space-x-4"
            />
          </div>
        </div>
        <div
          class="flex items-center"
        />
      </div>
    </nav>
  </header>
  <main>
    <div
      class="max-w-7xl mx-auto px-4 py-6 sm:px-0"
    >
      <div
        class="border-1 border-gray-400 rounded-lg px-10 py-10 bg-gray-100 shadow"
      />
    </div>
  </main>
</div>
`;
